app/presenters/any_link_presenter.rb

```diff
@@ -0,0 +1,92 @@
+class AnyLinkPresenter
+  def self.for(link_container, *args)
+    presenter_klass =
+      case link_container
+      when Link
+        ForALink
+      when LocalAuthority
```

Comment:
This whole presenter may have been brought over from the prototype prematurely.  In the prototype we wanted to present all links, including the home page link in the same list view, so it was useful to have a single object that would act appropriately given a `Link` or a `LocalAuthority`.  If we're not doing that anymore it might not be useful, and you could put the extra methods from here into `LinkPresenter`.